python/ray/experimental/channel/torch_tensor_nccl_channel.py

```diff
@@ -517,11 +538,19 @@ def _init_nccl_group(
 
 
 def _destroy_nccl_group(group_id: str) -> None:
+    """
+    Destroy the NCCL group with the given ID.
+
+    If this NCCL group is supplied by the user (not created internally
+    by the DAG), then this is a no-op.
+    """
     ctx = ChannelContext.get_current()
     if group_id not in ctx.nccl_groups:
         return
 
     group = ctx.nccl_groups[group_id]
+    if not isinstance(group, _NcclGroup):
```

Comment:
```suggestion
    if not isinstance(group, _NcclGroup):
```
Can we call `_NcclGroup` a `_DefaultNcclGroup`? Or add a comment here that _NcclGroup is the default one

Reply:
Added a comment in class definition.